### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by pedroagcf**

### CODE REVIEW

**Comment by pedroagcf** [comment]
Hey @ankitwww, I don't know what's going on here. I can't find the dev branch to send the PR. Could you please help me?

**Comment by ankitwww** [comment]
@pedroagcf - plz directly raise against main...

File: components/Footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Center, HStack } from "@chakra-ui/react";
+import { FaGithub, FaTwitter } from "react-icons/fa";
+import Link from "next/link";
+
+export const Footer = () => {
+  return (
+    <Center height={300} w="100%" bg="teal" gap={5}>
```
Comment (ankitwww) [inline components/Footer/index.tsx:7]: Height 300 may look odd on desktop view

**Comment by ankitwww** [comment]
Fixes #1

**Pull request merged**